src/data/bluetooth.ts

```diff
@@ -0,0 +1,34 @@
+import type { UnsubscribeFunc } from "home-assistant-js-websocket";
+import type { HomeAssistant } from "../types";
+import type { DataTableRowData } from "../components/data-table/ha-data-table";
+
+export interface DeviceRowData extends DataTableRowData {
+  address: string;
+  connectable: boolean;
+  manufacturer_data: { [id: number]: string };
+  name: string;
+  rssi: number;
+  service_data: { [id: string]: string };
+  service_uuids: string[];
+  source: string;
+  time: number;
+  tx_power: number;
+}
+
+export interface RemoveDeviceRowData {
+  address: string;
+}
+
+export interface EventDataMessage {
```

Comment:
Can you add Bluetooth to the interface names. Otherwise we can end up in TypeScript auto-complete hell.

Reply:
`BluetoothEventDataMessage`. Btw, can't we just call it `BluetoothEvent` ?